tests/Paket.Tests/Packaging/TemplateFileParsing.fs

```diff
@@ -163,6 +163,8 @@ LANGUAGE
     en-gb
 tags
     rop, fsharp F#
+packageTypes
+    DotnetTool, Template
```

Comment:
Do we need to normalize user input somehow here?

Reply:
no, i think is better to leave these as is.

Reply:
Yes I'm fine with that (was basically just asking what you think)

Reply:
Yes, i was thinking about it too, good question.to validate it too (to output the right word with the same case as nuget, to play safe).